cli/cfncluster/cfncluster.py

```diff
@@ -65,31 +64,36 @@ def create(args):
 
     capabilities = ["CAPABILITY_IAM"]
     try:
-        cfnconn = boto.cloudformation.connect_to_region(config.region,aws_access_key_id=config.aws_access_key_id,
-                                                        aws_secret_access_key=config.aws_secret_access_key)
+        cfn = boto3.client('cloudformation', region_name=config.region,
+                           aws_access_key_id=config.aws_access_key_id,
+                           aws_secret_access_key=config.aws_secret_access_key)
     except Exception as e:
         logger.critical(e.message)
```

Comment:
there is no "message" in e.
Should be
logger.critical(e.response.get('Error').get('Message'))
?